client/src/components/UI_LoadingWrapper.re

```diff
@@ -49,15 +54,13 @@ let make = (~delayMs=1000, ~loading: bool, children) => {
          * Switching from loading to not loading
          * Clean up and stop spinner
          */
-        clearTimeout(self.state.timeoutId^);
-        self.send(ChangeLoadingDisplay(false));
+        clearTimeout(timeoutId^);
+        dispatch @@ ChangeLoadingDisplay(false);
       };
+      None;
     },
-  reducer: (action, state) =>
-    switch (action) {
-    | ChangeLoadingDisplay(loading) =>
-      ReasonReact.Update({...state, displayLoading: loading})
-    | ChangeLoading(loading) => ReasonReact.Update({...state, loading})
-    },
-  render: ({state}) => children(state.displayLoading),
+    [|loading|],
+  );
+
+  displayLoading ? children : ReactOld.null;
```

Comment:
This is not the same behavior as before. It should be something like `children(displayLoading)`.

Reply:
Sorry, i am not sure `children` can be function. But maybe. Thanks!